echoshader/echogram.py

```diff
@@ -34,6 +35,8 @@ def single_echogram(
         Input list like ['#0000ff', '#00ffff'] to customize colormap.
     value_range : tuple[float, float]
         The minimum and maximum value for the color scale of the echogram.
+    vert_dim : str, optional
+        The name of the vertical dimension.
```

Comment:
The assumption is that it is the 1D dimension coordinate. Here I just allow it to be called either 'echo_range' or 'depth' or whatever the person has called their dimension (in case they renamed it). The idea is that it is coming from `compute_MVBS`.  I wanted to use it with these regridded datasets:
<img width="574" alt="Screenshot 2023-11-29 at 5 06 38 PM" src="https://github.com/OSOceanAcoustics/echoshader/assets/11621647/c0091566-504b-49f8-b41e-4d3dce73ba36">
Is it true that for `compute_MVBS` it is always 1D? Maybe that can be stressed more in the documentation, especially, for people trying on their own data, not using our notebook generation.

Reply:
> Is it true that for compute_MVBS it is always 1D?

Yes, the output would always be 1D.

How about adding this to the docstring here?
```suggestion
    vert_dim : str, optional
        The name of the vertical dimension, must be 1D.
```